src/Pages/Authpages/Login.jsx

```diff
@@ -1,74 +1,122 @@
-import {IoIosArrowDropright} from "../../components/Icons/Icons"
+import { IoIosArrowDropright } from "../../components/Icons/Icons";
 import { Link } from "react-router-dom";
+import { useRef, useEffect } from "react";
+import { useAuth } from "../../Context/AuthContext";
+import { useLocation,useNavigate } from "react-router-dom";
+import axios from "axios";
+
 const Login = () => {
+  const useremail = useRef();
+  const userpass = useRef();
+  const navigate = useNavigate();
+  const location = useLocation();
+  
+
+  const { isLogin, setLogin, Error,setError } = useAuth();
+
+  const loginHandler = async () => {
+    try {
+      const response = await axios.post(`/api/auth/login`, {
+        email: useremail.current.value,
+        password: userpass.current.value,
+      });
+      setLogin(true);
+      console.log(response);
+     
+      const userDetail = {
+        Email: response.data.foundUser.email,
+        name:
+          response.data.foundUser.firstName +
+          " " +
+          response.data.foundUser.lastName,
+      };
+      localStorage.setItem("token", response.data.encodedToken);
+      localStorage.setItem("user", JSON.stringify(userDetail));
+    } catch (error) {
+      console.log(error)
```

Comment:
- Working as intended to work
- ok to have console.logs now, but must remove them at the time of final production
- you can use built-in react-alert npm packages for showing alert messages at the time like -  error, success login etc (suggestion you can do it in later branch)
- code is clean and easy to understand